### ISSUE TITLE
Ekos not saving settings

### ISSUE THREAD

**Issue by mmichal12** [rkaczorek/astroberry-server#179]
After installing latest version of Astroberry my settings in Ekos (mount connection, IP address etc.) are not saved. This hasn't happened with previous version. It is fresh installation with all updates installed.

**Comment by rkaczorek** [comment]
Do you mean Ekos settings as seen in KStars or drivers settings as seen in INDI Panel?

**Comment by mmichal12** [comment]
INDI panel, sorry for confusion. But not only for one driver, in my case for the mount, NMEA GPS, not sure about DSLR now

**Comment by rkaczorek** [comment]
There is no autosave for drivers and never was ;-) You need to Save your parameters on each driver Options tab.

**Comment by mmichal12** [comment]
Yes, I tried, setting are not saved, it is always in default config. I tried to purge the config too, from logs I don' see anything helpful to me

**Comment by rkaczorek** [comment]
You might inspect or even remove all configuration files from /home/astroberry/.indi (note . in front of indi)
Before doing anything, check permisions to the config files by running `ls -al /home/astroberry/.indi`
If you decide to remove all configs run `rm -rf /home/astroberry/.indi`

**Comment by mmichal12** [comment]
ok, thanks, I did some checks and it looks the config file gets saved but doesn't get loaded automatically, when I click Load it loads the config but there is always default config when I start Ekos. Is this how it should be?

**Comment by rkaczorek** [comment]
You need to set Load device configuration / On connection in Ekos settings

**Comment by mmichal12** [comment]
yes, I have it set to Load configuration, but not loading, I have to click on Load for each device, which is still better than add the values there  every time

**Comment by bbillp** [comment]
I also discovered a similar problem with the latest KSTARS/EKOS version last night.
Open KSTARS, Open EKOS, Select the previously used profile for Celestron CGX, Nikon D5600 Imager, ASI120 guide.
The D5600 Image Options Tab did not save RAW, NEF, NATIVE .  Also the camera setting for FRAME, WIDTH 600 displays 600 even when attempting to set to 6000 but the camera still works OK, it is just the display that is wrong.

Thanks for the wonderful software, the PLATE Solving is fantastic .
